BenchmarkTests/Benchmarks/Sources/MetricExporter.swift

```diff
@@ -0,0 +1,162 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import Foundation
+import OpenTelemetrySdk
+
+enum MetricExporterError: Error {
+    case unsupportedMetric(aggregation: AggregationType, dataType: Any.Type)
+}
+
+/// Replacement of otel `DatadogExporter` for metrics.
+///
+/// This version does not store data to disk, it uploads to the intake directly.
+/// Additionally, it does not crash.
```

Comment:
Is this crash a known issue in `opentelemetry-swift`? If so let's link it here, so we can go back to the OOB exporter once fixed. If there are no reports, let's make one.

Reply:
I haven't reported it just yet, will do.
Basically, [these](https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/Exporters/DatadogExporter/Metrics/MetricsExporter.swift#L69-L84) force unwrap can fail.